pkgs/sdk/server/src/LaunchDarkly.ServerSdk.csproj

```diff
@@ -48,11 +48,10 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'net462'">
     <PackageReference Include="System.Text.Json" Version="6.0.0" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="8.0.0" />
```

Comment:
For reviewers: ServerSdk package doesn't need diagnostics.  This is in the ServerSdk.Telemetry package now.